### PR TITLE
Update local ci parameters to run locally

### PR DESCRIPTION
**Pull request by jamepark4**
Implement updates to run tempest locally through repo.

### CODE REVIEW

File: ci/nova-operator-compute-kit/topology/base/infra.yaml [hunk 1]
```diff
@@ -13,8 +13,14 @@ spec:
         storageRequest: 500M
   memcached:
     enabled: true
+    templates:
+      memcached:
+        replicas: 1
   ovn:
     template:
+      ovnController:
+        external-ids:
```
Comment (SeanMooney) [inline ci/nova-operator-compute-kit/topology/base/infra.yaml:22]: its proably the default but maybe add replica: 1

 ovn:
    template:
      ovnController:
        replicas:1
        external-ids: ...

File: README.md [hunk 2]
```diff
@@ -164,6 +164,36 @@ make manifests
...
+cat ~/my-envs.yml
+---
+nova_operator_basedir: '/home/stack/nova-operator'
+target_host: 'localhost'
+```
+
+4. Run tempest playbook
+```sh
+ansible-playbook ci/nova-operator-compute-kit/playbooks/tempest.yaml -e @${HOME}/my-envs.yml
+```
```
Comment (gibizer) [inline README.md:195]: This procedure works for me.

**Review by SeanMooney — APPROVED** [review]
thanks james

**Pull request merged**